nix/modules/blocksense/module-opts/anvil.nix

```
@@ -33,8 +33,9 @@ in
       description = "The fork URL to use for the Anvil instance.";
     };
 
```

Given that the port is a service option, why not make the ip/endpoint one as well, that defaults to http://127.0.0.1/?

```suggestion

ip = mkOption {
      type = types.str;
      default = "http://127.0.0.1/;
      default = 8544;
      description = "The ip to use for the Anvil instance.";
    };
```

---

Unrelated to this PR